### PR TITLE
Add copy, cut, paste options to edit menu

### LINKED ISSUE #207: add "copy", "paste", "cut" options to "edit"
Putting these commands into menu would show they are available. Now you have to know about keyboard shortcuts.

### PR DESCRIPTION
**Pull request by ppd-touk**
Closes #207

### CODE REVIEW

**Comment by jedrz** [comment]
I think it's good idea to provide a screenshot or gif how the FE look like after the changes, e.g. https://github.com/TouK/nussknacker/pull/94. WDYT?

**Comment by ppd-touk** [comment]
IAWT
edit

File: ui/client/containers/Visualization.js [hunk 1]
```diff
@@ -134,6 +152,76 @@ class Visualization extends React.Component {
     this.props.actions.deleteNodes(this.props.selectionState)
   }
 
+  copySelection = (event, shouldCreateNotification) => {
```
Comment (jedrz) [inline ui/client/containers/Visualization.js]: I think `shouldCreateNotification` is reduntant because it is always true.

Comment (jedrz) [reply ui/client/containers/Visualization.js]: Ahh, I see now - it's used for cut

**Comment by ppd-touk** [comment]
@jedrz Shortcuts like `ctrl+c`,` ctrl+x`, `cxtr+v` still work.

**Pull request merged**